controllers/suite_test.go

```diff
@@ -39,10 +38,7 @@ var testEnv *envtest.Environment
 
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
-
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+	RunSpecs(t, "Release Controller Suite")
```

Comment:
Any idea why were we using the custom reporter? Just for curiosity, then I can approve as it LGTM.